stepfunctions/stepfunctions/functions/create_alias/index.py

```diff
@@ -0,0 +1,31 @@
+import json
+import boto3
+import uuid
+
+
+def handler(event, context):
+    
+    bedrock_agent = boto3.client('bedrock-agent')
+    
+    agent_alias = str(uuid.uuid4())
+    agent_id = event["update_output"]["agentid"]
+    
+    alias_resp = bedrock_agent.create_agent_alias(
+    agentAliasName=agent_alias,
+    agentId=agent_id
+    )
+    
+    print(alias_resp)
```

Comment:
logger from the powertool produces cleaner logging format. If you decide to use powertools across all lambda functions, please make it consistent :)